includes/class-wc-stripe-blocks-support.php

```diff
@@ -60,6 +60,14 @@ public function is_active() {
 	 * @return array
 	 */
 	public function get_payment_method_script_handles() {
+		if ( is_product() && ! WC_Stripe_Helper::should_load_scripts_on_product_page() ) {
+			return;
+		}
```

Comment:
I actually don't think this is necessary. We don't load the blocks on the product page.

Probably better to just change the cart check to see if the `woocommerce/cart` block is present and remove the `is_product()` check entirely 🤔

Reply:
Based on https://github.com/woocommerce/woocommerce-gateway-stripe/pull/2110#issuecomment-962416878 we're going to always load the scripts in the cart block. It's loaded asynchronously so it shouldn't have a big impact on load times.